Brightness control is currently broken with elogind, so fall back to
the previous codepath which doesn't use elogind.
See https://github.com/elogind/elogind/issues/158
diff --git a/plugins/power/gsd-backlight.c b/plugins/power/gsd-backlight.c
index ca5f272..59048d9 100644
--- a/plugins/power/gsd-backlight.c
+++ b/plugins/power/gsd-backlight.c
@@ -482,7 +482,7 @@ gsd_backlight_set_brightness_val_async (GsdBacklight *backlight,
         if (backlight->udev_device != NULL) {
                 BacklightHelperData *task_data;
 
-                if (backlight->logind_proxy) {
+                if (0) {
                         g_dbus_proxy_call (backlight->logind_proxy,
                                            "SetBrightness",
                                            g_variant_new ("(ssu)",
